### PR TITLE
chore: display params for catalog

### LINKED ISSUE #822: Support viewing parameters for catalogs, groups & controls
### Description

As a user of the OSCAL Viewer/Editor, I would like to be able to view `params `for catalogs, groups, and controls.

### Acceptance Criteria

* [`params`](https://pages.nist.gov/OSCAL/reference/1.0.5/complete/json-reference/#/catalog/controls/params) are properly displayed at the catalog, group, and control levels
  * This includes presentation of the list of parameters, to include: `label`, `usage`, `constraint`, `guideline`, `value`/`select`, and `remarks`

### Proposed Solution

Ideally params is made into it's own component that is passed to catalogs, groups, and controls to be displayed.

### Additional Notes

_No response_

### Implementation

- [ ] I can (or plan to) submit a pull request to implement this
- [ ] Implementing this may result in a breaking change

### PR DESCRIPTION
**Pull request by tuckerzp**
Displays parameters at the top of a catalog.

closes #822 

### Testing

https://raw.githubusercontent.com/EasyDynamics/oscal-demo-content/tuckerzp/params/catalogs/basic-test-catalog.json

image

### CODE REVIEW

File: packages/oscal-react-library/src/components/OSCALCatalog.tsx [hunk 1]
```diff
@@ -81,6 +83,9 @@ export const OSCALCatalog: React.FC<OSCALCatalogProps> = ({
 
   return (
     <OSCALDocumentRoot>
+      <Stack justifyContent="flex-end" direction="row">
+        <OSCALParamsDialog params={catalog.params} />
+      </Stack>
```
Comment (kylelaker) [inline packages/oscal-react-library/src/components/OSCALCatalog.tsx:88]: It'd rather not hide this behind a dialog here -- I would almost prefer to display the dialog contents directly. Can we get feedback from the team and @brian-easyd on this?

Comment (tuckerzp) [reply packages/oscal-react-library/src/components/OSCALCatalog.tsx:88]: Good point, it would be easy enough to switch that.

Comment (tuckerzp) [reply packages/oscal-react-library/src/components/OSCALCatalog.tsx:88]: With Dialog

image

---

Without Dialog

image

Comment (kylelaker) [reply packages/oscal-react-library/src/components/OSCALCatalog.tsx]: Can you show with 800-53?

Comment (tuckerzp) [reply packages/oscal-react-library/src/components/OSCALCatalog.tsx]: image

**Review by kylelaker — APPROVED** [review]

**Review by Bronstrom — APPROVED** [review]

**Pull request merged**